### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by tkchin**
Log GCM server failures.
Make only one POST request to GCM server.
Fix version_info.json not found error on unit tests and in chrome app.

### CODE REVIEW

**Comment by tkchin** [comment]
@jiayliu @chuckhays PTAL

File: src/app_engine/gcm_notify.py [hunk 1]
```diff
@@ -49,62 +49,46 @@ def get_gcm_api_key():
...
   }
+  payload = create_gcm_payload(gcm_ids, collapse_key, message)
+  start_time = time.time()
+  # TODO(tkchin): investigate setting follow_redirects=False.
+  result = urlfetch.fetch(url=GCM_API_URL,
+                          payload=payload,
+                          method=urlfetch.POST,
+                          headers=headers)
+  end_time = time.time()
+  logging.info('Took %.3fs to send GCM message request.', end_time - start_time)
```
Comment (jiayliu) [inline src/app_engine/gcm_notify.py]: nit: log the length of gcm_ids

**Comment by jiayliu** [comment]
lgtm % nit
Could you port the version_info fix to Master?

**Comment by tkchin** [comment]
There doesn't seem to be problems with version_info in master for some reason (no errors on unit tests). Still want the port?

**Comment by jiayliu** [comment]
Fine to keep as it is then.

On Tue, Feb 10, 2015 at 12:33 PM, Zeke Chin <notifications@github.com>
wrote:

> There doesn't seem to be problems with version_info in master for some
> reason (no errors on unit tests). Still want the port?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/54#issuecomment-73778036>.
>



-- 
Jiayang Liu

**Comment by tkchin** [comment]
rebased to current. PTAL

**Comment by jiayliu** [comment]
lgtm

**Comment by chuckhays** [comment]
lgtm too

**Pull request merged**